config/webpack.chart-renderer.config.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-env node*/
+/*eslint import/no-nodejs-modules:0 */
+const baseConfig = require('../webpack.config');
+
+const config = {
+  ...baseConfig,
+
+  target: 'node',
+  entry: {
+    chartRendererConfig: 'app/components/charts/backendStyles',
+  },
+
+  optimization: {},
+
+  output: {
+    ...baseConfig.output,
+    libraryTarget: 'commonjs',
```

Comment:
Are you not able to us es-modules?

Reply:
I wouldn't mind, but right now older node doesn't understand esmodules (this file is loaded into the chart renderer)

Reply:
Do we have older node?

Reply:
We're using node 14 now for this service.

I'll take a look to see if I can get this to compile a modules version 👏